File: src/main/scala/org/cyclops/fluidconverters/ClientProxy.scala
package org.cyclops.fluidconverters

import cpw.mods.fml.client.registry.ClientRegistry
import org.cyclops.fluidconverters.render.RenderFluidConverter
import org.cyclops.fluidconverters.tileentity.TileEntityFluidConverter

/**
 * Client proxy.
 * @author rubensworks
 */
class ClientProxy extends CommonProxy {

    override def registerRenderers() {
        ClientRegistry.bindTileEntitySpecialRenderer(classOf[TileEntityFluidConverter], RenderFluidConverter)
    }

}

File: src/main/java/org/cyclops/fluidconverters/proxy/ClientProxy.java
package org.cyclops.fluidconverters.proxy;

import net.minecraftforge.common.MinecraftForge;
import org.cyclops.cyclopscore.init.ModBase;
import org.cyclops.cyclopscore.proxy.ClientProxyComponent;
import org.cyclops.fluidconverters.FluidConverters;
import org.cyclops.fluidconverters.event.TextureStitchEventHook;

/**
 * Proxy for the client side.
 * 
 * @author rubensworks
 * 
 */
public class ClientProxy extends ClientProxyComponent {

	public ClientProxy() {
		super(new CommonProxy());
	}

	@Override
	public ModBase getMod() {
		return FluidConverters._instance;
	}

	@Override
	public void registerEventHooks() {
		super.registerEventHooks();
		MinecraftForge.EVENT_BUS.register(new TextureStitchEventHook());
	}
}
